discovery-provider/alembic/versions/436c10e54758_create_profile_challenges.py

```diff
@@ -24,7 +24,8 @@ def upgrade():
 
     # Create the challenges if needed
     session = Session(bind=connection)
-    create_new_challenges(session)
```

Comment:
ooh I did a fix for this in my outstanding PR, but this is much cleaner CC: @piazzatron